packages/moonstone/styles/variables.less

```diff
@@ -470,7 +470,7 @@
 @moon-notification-padding-left-right: 39px;
 @moon-notification-padding-top-bottom: 24px;
 @moon-notification-min-width: 300px;
-@moon-notification-max-width: 978px;
+@moon-notification-max-width: 1017px;
```

Comment:
Can we calculate this from the max button width?

Reply:
There are other values involved. Ex: The previous value was not `300px * 3`, it's `978px`. A math based solution would be nice, but we need to understand the other sizing influencers.